react/react2/react2-week1/app/blogs/[blogId]/page.js

```diff
@@ -0,0 +1,13 @@
+"use client";
+import { usePathname } from "next/navigation";
+
+export default function Blog() {
+  const path = usePathname();
+  const lastPartofPath = path?.split("/").pop();
+  const formatWeWant = (word) => {
+    return word
+      .replace(/-/g, " ")
+      .replace(/\b\w/g, (char) => char.toUpperCase()); // Capitalize the first letter of each word;
```

Comment:
Just a performance tip for the future: every time your components rerenders, you will have new `formatWeWant` function. But it is not really bound to anything connected to this component, so we can optimise it a little bit by moving it outside of component

So instead of
```
function Blog() {
...
const formatWeWant = () => {}
...
}
```
do something like
```
const formatWeWant = () => {}

function Blog() {
...
}
```